### ISSUE TITLE
Issue using launcher after installing with homebrew (Apple M1)

### ISSUE THREAD

**Issue by Meekohi** [NethermindEth/nethermind#4495]
**Describe the bug**
Homebrew installation for macOS 12.5.1 (Apple M1) doesn't seem to work. `Nethermind.Runner` doesn't exist in the homebrew installation as best I can tell (should be `./bin/nethermind` maybe?)

**To Reproduce**
```
brew tap nethermindeth/nethermind
brew install nethermind
nethermind-launcher

? Start Nethermind Ethereum Node
? Select network Ethereum (mainnet)
? Select sync Fast sync

There was an error when starting ./Nethermind.Runner
Error: spawn ./Nethermind.Runner ENOENT
```

nethermind 1.14.0
From: https://github.com/nethermindeth/homebrew-nethermind/blob/HEAD/nethermind.rb

see https://github.com/NethermindEth/homebrew-nethermind/blob/d9c12ff20c3025a1f1a0226619148c74110b0f6e/nethermind.rb#L22-L24 where the renaming happens.

**Comment by tkstanczak** [comment]
@FalcoXYZ ?

**Comment by Meekohi** [comment]
@FalcoXYZ Yes, running `nethermind` directly works after doing the `codesign` above 👍🏼

**Comment by FalcoXYZ** [comment]
@Meekohi Would it be possible to try running `nethermind` directly without the codesign commands? Just so we can check if that was the issue or not? 

You can remove and reinstall nethermind by using the following commands: 

```
brew rm nethermind
brew install nethermind
```

**Comment by Meekohi** [comment]
Yes I tried it before codesign and it did not work (immediately was killed with no error message)

**Comment by FalcoXYZ** [comment]
Can you please confirm whether or not the codesign fixes the issues you were facing? 
Are nethermind, nethermind-launcher and nethermind-cli broken for you before the codesign and do they work after using the codesign commands? Just want to double-check! I'm already checking for a possible solution :)

**Comment by Meekohi** [comment]
No worries @FalcoXYZ -- I think the code-signing and ENOENT issues are two independent problems.

After codesigning, there is still the issue of
```
➜  ~ nethermind-launcher 
? Start Nethermind Ethereum Node
? Select network Ethereum (mainnet)
? Select sync Fast sync
There was an error when starting ./Nethermind.Runner
Error: spawn ./Nethermind.Runner ENOENT
```

Which is simply because it is trying to run `/Nethermind.Runner` but it has been renamed to `nethermind`.

**Comment by kamilchodola** [comment]
@brbrr didn't you had that on your end as well?

**Comment by stdevMac** [comment]
Closing: nethermind-launcher no longer exists and the Homebrew formula was rewritten to install the plain binary, so this report has no remaining target.
